packages/odyssey-storybook/src/components/_internal/TokenTables.tsx

```diff
@@ -0,0 +1,240 @@
+      ?.values.sort((a: Token, b: Token) => {
+        const nameA = a.name.toUpperCase();
+        const nameB = b.name.toUpperCase();
+        if (nameA < nameB) {
+          return -1;
+        }
+        if (nameA > nameB) {
+          return 1;
+        }
+        return 0;
+      });
+
+    setTables(tempTables);
+  }, [setTables]);
+
+  const renderBorder = (token: Token) => (
+    <span
+      style={{
+        width: "1.25em",
+        height: "1.25em",
+        borderWidth: token.name.includes("BorderWidth") ? token.value : "1px",
+        borderStyle: token.name.includes("BorderStyle")
+          ? (token.value as string)
+          : "solid",
+        borderColor: "#d7d7dc",
+        borderRadius: `${
+          token.name.includes("BorderRadius") ? token.value : ""
+        }`,
+        display: "inline-block",
+        verticalAlign: "middle",
+      }}
+    ></span>
+  );
+
+  const renderColor = (val: string | number) => (
+    <span
+      style={{
+        width: "1em",
+        height: "1em",
+        display: "inline-block",
+        backgroundColor: `${val}`,
+        borderStyle: "solid",
+        borderColor: `${val === "#ffffff" ? "#d7d7dc" : "transparent"}`,
+        borderRadius: "4px",
+        verticalAlign: "middle",
+      }}
+    ></span>
+  );
+
+  const renderColorText = (val: string | number) => (
+    <span
+      style={{
+        color: `${val}`,
+        display: "inline-block",
+        paddingInline: "4px",
+        backgroundColor: `${val === "#ffffff" ? "#000" : ""}`,
+      }}
+    >
+      Aa
+    </span>
+  );
+
+  const renderSpace = (val: string | number) => (
+    <span
+      style={{
+        width: `${val}`,
+        height: `${val}`,
+        display: "inline-block",
+        backgroundColor: "#f88c90",
+      }}
+    ></span>
+  );
+
+  const renderFont = (token: Token) => (
+    <span
+      style={{
+        fontFamily: `${token.name.includes("FontFamily") ? token.value : ""}`,
+        fontSize: `${token.name.includes("FontSize") ? token.value : ""}`,
+        fontWeight: `${
+          token.name.includes("FontWeight") ? token.value : ""
+        }` as unknown as FontWeight,
+        lineHeight: `${token.name.includes("LineHeight") ? token.value : ""}`,
+        backgroundColor: `${
+          token.name.includes("LineHeight") ? "#ebebed" : ""
+        }`,
+        display: `${token.name.includes("LineHeight") ? "inline-block" : ""}`,
+      }}
+    >
+      Waltz, bad nymph, for quick jigs vex!
+    </span>
+  );
+
+  const renderFocus = (val: string | number) => (
+    <span
+      style={{
+        width: "1.25em",
+        height: "1.25em",
+        display: "inline-block",
+        boxShadow: `0 0 0 ${val} #a7b5ec`,
+      }}
+    ></span>
+  );
+
+  const renderShadow = (val: string | number) => (
+    <span
+      style={{
+        width: "1.5em",
+        height: "1.5em",
+        display: "inline-block",
+        boxShadow: `${val}`,
+      }}
+    ></span>
```

Comment:
Out of curiosity, why not use a Box here?

Reply:
That was my initial stab, but the values that we get from the theme are absolute and the props that box wants aren't. So there's not much utility that box offers in this case. eg: here `val = "0 2px 0 rgba(175, 175, 175, 0.12)"` but box wants `boxShadow = "default"`